### PR TITLE
Flask webhook

### PR DESCRIPTION
**Pull request by VaishnaviHire**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from slack import WebClient
+from flask import Flask, request, make_response
+from config.config import bot_token
```
Comment (PratikMahajan) [inline app.py]: bot token in environment varibles. Need ENV for deployment

File: config/config.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import os
```
Comment (PratikMahajan) [inline config/config.py]: is this file needed ?

Comment (VaishnaviHire) [reply config/config.py]: yes. It makes it easier to update when configs change
